include/koa/authentication/passport-strategies/strategy_services.js

```diff
@@ -0,0 +1,93 @@
+function _getDao(loginContext, pb) {
+    if (loginContext.site) {
+        return new pb.SiteQueryService({
```

Comment:
I dont know if there is a difference between SiteQueryService with an onlyThisSite flag of false and DAO.  hm.  I think it picks site records over non site records if dupes are found?  

SQS > DAO 90% of the time... soo we rarely need to use actual DAO but this is core PB code so backwards compatibility with single tenant solutions should be maintained.